tests/WE2E/setup_WE2E_tests.sh

```diff
@@ -35,7 +35,7 @@ function usage {
 
 }
 
-machines=( hera jet )
+machines=( hera jet cheyenne orion wcoss2 gaea odin singularity macos noaacloud )
```

Comment:
We do only want these for hera and jet. A different mechanism tests the WE2E on the other platforms.

Reply:
I needed the script to do the same test as `hera` on `orion` even though we do not have CI enabled there. I recall having to do something similar in other systems to run the same set of fundamental tests manually.